stix_shifter_modules/secretserver/stix_transmission/status_connector.py

```diff
@@ -0,0 +1,50 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_status_connector import BaseStatusConnector
+from stix_shifter_utils.modules.base.stix_transmission.base_status_connector import Status
+from enum import Enum
+import json
+from stix_shifter_utils.utils.error_response import ErrorResponder
+
+class SecretServerStatus(Enum):
+    # WAIT, EXECUTE, SORTING, COMPLETED, CANCELED, ERROR
+    WAIT = 'WAIT'
+    EXECUTE = 'EXECUTE'
+    SORTING = 'SORTING'
+    COMPLETED = 'COMPLETED'
+    CANCELED = 'CANCELED'
+    ERROR = 'ERROR'
+
+
+class StatusConnector(BaseStatusConnector):
+    def __init__(self, api_client):
+        self.api_client = api_client
+
+    def __getStatus(self, status):
+        switcher = {
+            SecretServerStatus.WAIT.value: Status.RUNNING,
+            SecretServerStatus.EXECUTE.value: Status.RUNNING,
+            SecretServerStatus.SORTING.value: Status.RUNNING,
+            SecretServerStatus.COMPLETED.value: Status.COMPLETED,
+            SecretServerStatus.CANCELED.value: Status.CANCELED,
+            SecretServerStatus.ERROR.value: Status.ERROR
+        }
+        return switcher.get(status).value
+
+    def create_status_connection(self, search_id):
+        # Grab the response, extract the response code, and convert it to readable json
+        # Verify the input
+        response = self.api_client.get_status(search_id)
+        response_code = response.code
+        response_dict = json.loads(response.read())
+
+        # Construct a response object
+        return_obj = dict()
+        response_code = 200
+        if response_code == 200:
+            return_obj['success'] = True
+            return_obj['status'] = self.__getStatus(response_dict['status'])
```

Comment:
not sure the purpose of `self.__getStatus(response_dict['status'])` function. `self.api_client.get_status(search_id) `only returns `COMPLETED`